### ISSUE TITLE
DELETE: multiple vertices

### ISSUE THREAD

**Issue by deividasjackus** [orientechnologies/orientdb#5576]
How has https://github.com/orientechnologies/orientdb/issues/3177 been solved?

```
orientdb {db=test}> DELETE VERTEX [#13:2, #13:3]
```

> Error: com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error on parsing command at position #0: Class '[#13:2, #13:3]' was not found

```
orientdb {db=test}> DELETE VERTEX FROM [#13:2, #13:3]
```

> Error: com.orientechnologies.orient.core.command.OCommandExecutorNotFoundException: Cannot find a command executor for the command request: sql.

```
orientdb {db=test}> DELETE VERTEX FROM (SELECT FROM [#13:2, #13:3])
```

> Delete record(s) '2' in 0.015000 sec(s).

**Comment by luigidellaquila** [comment]
Hi @sinfex 

it seems to be a different problem, but the solution should be easy. I'll let you know in short

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @sinfex 

This issue was fixed long time ago in V 3.0

Closing, thanks

Luigi
